lib/features/game_config/bloc/game_config_cubit.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:bloc/bloc.dart';
+import 'package:kahootify/core/models/api/category.dart';
+import 'package:kahootify/features/game_config/models/game_config.dart';
+
+class GameConfigCubit extends Cubit<GameConfig> {
+  GameConfigCubit() : super(GameConfig());
+
+  void setGameName(String gameName) => emit(state.copyWith(gameName: gameName));
+
+  void setMaxNumberOfPlayers(int maxNumberOfPlayers) => emit(state.copyWith(maxNumberOfPlayers: maxNumberOfPlayers));
+
+  void setChosenCategory(Category chosenCategory) => emit(state.copyWith(chosenCategory: chosenCategory));
+
+  void setTimeForAnswer(int timeForAnswer) => emit(state.copyWith(timeForAnswer: timeForAnswer));
```

Comment:
I promised not to upset you with bad names but setAnswerTimeLimit would be better